src/js/components/VideoPage.js

```diff
@@ -71,23 +73,27 @@ class VideoPage extends React.Component {
           onVideoEnd={this.handleVideoEnd}
           playAgain={this.state.playAgain}
         />
-        { videoEnded &&
-          <AfterVideoCard />
-        }
-        <div className="buttons-container">
-          {fscreen.fullscreenEnabled && !videoEnded &&
-            <button onClick={this._setFullscreen} className="button">
-              GO FULLSCREEN
-            </button>
-          }
-          {videoEnded &&
-            <Fragment>
-              <button onClick={this._playAgain} className="button">
+        {IS_DEFAULT_MODE && (
+          <Fragment>
+            { videoEnded &&
+              <AfterVideoCard />
+            }
+            <div className="buttons-container">
+              {fscreen.fullscreenEnabled && !videoEnded &&
+                <button onClick={this._setFullscreen} className="button">
+                GO FULLSCREEN
+                </button>
+              }
+              {videoEnded &&
+              <Fragment>
```

Comment:
Unnecessary Fragment with one child
```suggestion
{videoEnded &&
  <button onClick={this._playAgain} className="button">
    PLAY AGAIN
  </button>
```

Reply:
has 2 children, the `div` and the `AfterVideoCard` component